### PR TITLE
Fix staking modals

### PR DESCRIPTION
**Pull request by kojiadrianojr**
Disable Claim action
image

Fix modals 
Farms modals

### CODE REVIEW

File: src/views/Pools/components/ClaimAction/index.tsx [hunk 1]
```diff
@@ -11,19 +11,19 @@ const ClaimAction: React.FC = () => {
...
+          MGG
         </Text>
         <Text bold textTransform='uppercase' color='textSubtle' fontSize='12px'>
           Earned
         </Text>
       </ActionTitles>
       <ActionContent>
         <div>
-          <Earned>123</Earned>
+          <Earned>0</Earned>
```
Comment (medardm) [inline src/views/Pools/components/ClaimAction/index.tsx:22]: nata naka static

**Review by medardm — CHANGES_REQUESTED** [review]
pa dynamic pls ng earned value

**Review by swengr-janan — APPROVED** [review]
Dynamite!

File: src/views/Pools/components/ClaimAction/index.tsx [hunk 2]
```diff
@@ -1,30 +1,94 @@
...
+    let displayBalance = userDataReady ? earnings.toLocaleString() : <Skeleton width={60} />
+
+    // If user didn't connect wallet default balance will be 0
+    if (!earningsBigNumber.isZero()) {
+      earnings = getBalanceAmount(earningsBigNumber)
+      earningsBusd = earnings.multipliedBy(cakePrice).toNumber()
+      displayBalance = earnings.toFixed(3, BigNumber.ROUND_DOWN)
+    }
+    const [pendingTx, setPendingTx] = useState(false)
+    const { onReward } = useClaim(stakingContract)
```
Comment (medardm) [inline src/views/Pools/components/ClaimAction/index.tsx:46]: Do not reuse functions from LP staking that is not compatible in Pool-based

**Review by medardm — CHANGES_REQUESTED** [review]

**Review by medardm — APPROVED** [review]
LGTM

**Pull request merged**